Makefile

```diff
@@ -306,23 +306,19 @@ verify: vendor verify-codegen verify-mockgen verify-manifests #HELP Run all veri
 
 #SECTION Release
 
-.PHONY: pull-opm
-pull-opm:
-	docker pull $(OPERATOR_REGISTRY_IMAGE)
-
 .PHONY: package
 package: $(YQ) $(HELM) #HELP Package OLM for release
-package: OLM_RELEASE_IMG_REF=$(shell docker inspect --format='{{index .RepoDigests 0}}' $(IMAGE_REPO):$(RELEASE_VERSION))
-package: OPM_IMAGE_REF=$(shell docker inspect --format='{{index .RepoDigests 0}}' $(OPERATOR_REGISTRY_IMAGE))
+package: OLM_RELEASE_IMG_REF=$(shell go run util/image-canonical-ref/main.go $(IMAGE_REPO):$(RELEASE_VERSION))
+package: OPM_IMAGE_REF=$(shell go run util/image-canonical-ref/main.go $(OPERATOR_REGISTRY_IMAGE))
 package:
 ifndef TARGET
 	$(error TARGET is undefined)
 endif
 ifndef RELEASE_VERSION
 	$(error RELEASE_VERSION is undefined)
 endif
-	@echo "Getting operator registry image"
-	docker pull $(OPERATOR_REGISTRY_IMAGE)
```

Comment:
We no longer need to run `docker pull` in order to resolve the canonical digest so I removed this `docker pull`, one below and `pull-opm` target (no used anywhere in the scripts and I doubt that it is being used by humans).